### PR TITLE
Double load check on ppc64 compute nodes

### PR DESCRIPTION
**Pull request by ramereth**
NOTE: I ran this manually on a Fedora ppc64 TK run to verify this works as expected and confirmed it did.

### CODE REVIEW

**Comment by ramereth** [comment]
@osuosl-cookbooks/chefs ready for review

File: test/integration/mon/serverspec/mon_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require 'serverspec'
+
+set :backend, :exec
+
+load_thres = if %w(ppc64 ppc64le).include?(os[:arch])
+               '-w 14,9,4 -c 18,13,8'
```
Comment (jldugger) [inline test/integration/mon/serverspec/mon_spec.rb]: This works for openpower1, but 5 and 6 are substantially more burdened:

http://munin2.osuosl.org/osuosl.org/openpower1.osuosl.org/load.html
http://munin2.osuosl.org/osuosl.org/openpower5.osuosl.org/load.html
http://munin2.osuosl.org/osuosl.org/openpower6.osuosl.org/load.html

Their long term average is 10, with regular spikes up to 30.

Comment (jldugger) [inline test/integration/mon/serverspec/mon_spec.rb]: Ah, I guess this is for the test kitchen, where cpu count would probably be lower.

**Comment by jldugger** [comment]
:+1:

File: test/integration/mon/serverspec/mon_spec.rb (hunk 1 above)
Comment (ramereth) [inline test/integration/mon/serverspec/mon_spec.rb]: Correct, these values are set based on the test kitchen environment. The ``total_cpu`` value for the nodes are 20, so using that you can see that the values should become ``-w 90,85,80 -c 170,165,160`` on the openpower nodes.

Is that acceptable?

Comment (jldugger) [inline test/integration/mon/serverspec/mon_spec.rb]: Yea, that's a fine formula for openpower machines.

**Pull request merged**